tex/paper.tex

```diff
@@ -158,20 +157,21 @@ \section*{Introduction}
     contribute new and original Iterated Prisoner's Dilemma strategies
 \end{itemize}
 
-The presented library is partly motivated by an ongoing discussion in the academic community
-about reproducible research \cite{Crick2014a, Hong2015a, Prlic2012, Sandve2013},
-and is:
+The presented library is partly motivated by an ongoing discussion in the
+academic community about reproducible research \cite{Crick2014a, Hong2015a,
+Prlic2012, Sandve2013}, and is:
 
 \begin{itemize}[noitemsep,topsep=0pt]
     \item Open: all code is released under an MIT license;
     \item Reproducible and well-tested: at the time of writing there is an excellent level of
-        integrated tests with 99.37\% coverage (including property based tests:
-        \cite{Hypothesis3.0.3}
+        integrated tests with 99.73\% coverage (including property based tests:
+        \cite{Hypothesis3.0.3})
     \item Well-documented: all features of the library are documented for ease of
         use and modification
-    \item Extensive: 123 strategies are included, with infinitely-many
-        strategies available in the case of parametrised strategies
-    \item Extensible: easy to modify to include new strategies and to run new tournaments
+    \item Extensive: 135 strategies are included, with infinitely-many
+        available in the case of parametrised strategies
```

Comment:
This might come up at the editorial/proofing stage. It's happened to me before at that stage where all things went one way or the other... I'm again indifferent...